src/character.py

```diff
@@ -45,10 +55,26 @@ def main():
     clock = pygame.time.Clock()
 
     running = True
+    jumping = 0
+    jumping_counter = 0
     while running:
         for event in pygame.event.get():
             if event.type == pygame.QUIT:
                 running = False
+        keys = pygame.key.get_pressed()
+        if keys[pygame.K_UP]:
```

Comment:
Though it's fine for now, I think we should consider having controls in a separate function just to keep things organized as we add more.